Assets/VRM/UniVRM/Editor/Tests/UniVRMSerializeTests.cs

```diff
@@ -231,7 +253,12 @@ public void MaterialTest()
             };
             var json2 = JsonSchema.FromType<glTF_VRM_Material>().Serialize(model, c);
             // NOTE: New serializer outputs values which will not be used...
-            Assert.AreEqual(json,json2);
+            Assert.AreEqual(json, json2);
+
+            // deserialize
+            var deserialized = default(glTF_VRM_Material);
+            json.ParseAsJson().Deserialize(ref deserialized);
+            Assert.AreEqual(1, deserialized.floatProperties.Count);
```

Comment:
I think that element values should be checked not only checking number of elements. Same comments for other fields.